packages/neon/neon/lib/src/blocs/user_statuses.dart

```diff
@@ -0,0 +1,87 @@
+part of '../../neon.dart';
+
+abstract class UserStatusesBlocEvents {
+  void load(final String username, {final bool force = false});
+}
+
+abstract class UserStatusesBlocStates {
+  BehaviorSubject<Map<String, Result<NextcloudUserStatusPublicStatus?>>> get statuses;
+}
+
+class UserStatusesBloc extends InteractiveBloc implements UserStatusesBlocEvents, UserStatusesBlocStates {
+  UserStatusesBloc(
+    this._platform,
+    this._account,
+  ) {
+    unawaited(refresh());
+    TimerBloc().registerTimer(const Duration(minutes: 5), refresh);
+  }
+
+  final NeonPlatform _platform;
+  final Account _account;
+  late Timer _timer;
```

Comment:
Does not look like this gets assigned somewhere.

Reply:
Oops forgot to remove that